.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go-version: [1.14.x, 1.15.x]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+
+    - name: Set up Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get Go dependencies
+      run: go get -v -t -d ./...
+
+    - name: Build
+      run: |
+        export PATH=~/go/bin:$PATH
+        make build
+
+    - name: Unit tests
+      run: go test ./...
```

Comment:
Setting up GH actions during initial project setup? Fancy!

Reply:
😉 except it didn't work 🙈